data-reconciliation-app/README.md

```diff
@@ -28,17 +28,25 @@ To get started run `cd data-reconciliation-app && make test` to run the applicat
 ├── governance
 │   └── constitution    CCF network constitution files
 │   └── vote            Contains proposal voting acceptance and rejection logic
-│   └── nodes           CCF network nodes config and docker files
+│   └── config          CCF network nodes configs
 │   └── scripts         All governance scripts
 │
-└── scripts             All the scripts to test, demo and deploy the application
 └── src                 Application source code
-    └── endpoints       Application endpoints implementation
+│    └── endpoints      Application endpoints implementation
+│
+└── test
+│    └── unit-test      Application unit tests
+│    └── e2e-test       Application end to end tests
+│
+└── deploy              Scripts to deploy to the application
+└── demo                Scripts to run demo
+└── docker              CCF network nodes docker files
```

Comment:
by having a folder with the dockerfile in it you will not be able to ../ inside a docker file - just remove this folder

Reply:
done